.github/workflows/integrations.yml

```diff
@@ -40,37 +44,48 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby-version }}
           bundler-cache: true
-          bundler: 1.17
 
       - name: "Cache gems"
-        uses: actions/cache@v2
+        uses: actions/cache@v3
         id: gem-cache
         with:
           path: vendor/bundle
           key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
           restore-keys: |
             ${{ runner.os }}-gems-
 
-      - name: Pre-run inline scripts
+      - name: Setup files
         run: |
           sudo mkdir -p /var/lib/rmt
           ruby -e "require 'yaml'; puts({'database_test'=>{'host' => '127.0.0.1', 'port' => 3306, 'username'=>'rmt','password'=>'rmt','database'=>'rmt_test','adapter'=>'mysql2','encoding'=>'utf8','timeout'=>5000,'pool'=>5}}.to_yaml)" > config/rmt.local.yml
+          gem env
+          bundler_version=$(bundler -v | cut -d ' ' -f 3)
+          echo "Ruby version: $(ruby -v)"
+          echo "Available bundler: $(gem list | grep bundler)"
+          # dropping default bundler setting
+          rm /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/specifications/default/bundler-*.gemspec
```

Comment:
What is this patch and why does it needs to get removed. Is `/opt/hostedtoolcache` the bundler work dir path?

Reply:
I tried to force remove  the default bundler version, but that code is dropped now from the branch.